packages/lego-bricks/src/components/Layout/Page/PageCover.module.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  grid-area: cover;
+  box-sizing: border-box;
+  max-width: var(--lego-max-width);
+  max-height: 330px;
+  aspect-ratio: 3.3;
+  width: 100%;
+  overflow: hidden;
+  position: relative;
+  background-color: var(--lego-card-color);
+
+  & img {
+    object-fit: cover;
+    display: block;
+    background: white;
```

Comment:
This does not look good on dark-mode

```suggestion
```

Reply:
Good catch! I believe it should be `var(--color-white)`, which is black in dark-mode

Reply:
That still does not match the rest of the page when on dark mode. 

It looks like `var(--lego-card-color)` matches the color for both light and dark mode, but I think the best solution is just not having a background color for `img`.